cypress/e2e/scenario-1-environment.cy.js

```diff
@@ -411,17 +411,15 @@ describe("Environment", () => {
     cy.get('[aria-label="Row-environment_agent_trigger_method"]')
       .find(".pf-c-form-control")
       .click();
-    cy.get(".pf-c-select__menu-item")
-      .contains("push_incremental_deploy")
-      .click();
+    cy.get(".pf-c-select__menu-item").contains("push_full_deploy").click();
     cy.get('[aria-label="Warning"]').should("exist");
     cy.get('[aria-label="Row-environment_agent_trigger_method"]')
       .find('[aria-label="SaveAction"]')
       .click();
     cy.get('[aria-label="Warning"]').should("not.exist");
     cy.get('[aria-label="Row-environment_agent_trigger_method"]')
       .find(".pf-c-form-control")
-      .should("have.value", "push_incremental_deploy");
+      .should("have.value", "push_full_deploy");
```

Comment:
Was the default value also set in iso-5 after all ? Thought it was only on iso-6 and plus. 
The issue I had to resolve specifically was about a clicking order. 

<img width="985" alt="image" src="https://user-images.githubusercontent.com/44098050/220052760-1d64e15c-fcc2-4274-a6cd-defd9f21212c.png">

Reply:
The issue here was that after cypress clicks on "push_incremental_deploy" there is no warning sign so it can't proceed further